### ISSUE TITLE
Metadata should be stripped from the gist and not the note

### ISSUE THREAD

**Issue by StephenCoady** [timrogers/obsidian-share-as-gist#400]
Hi,

When sharing as a gist and the metata option is turned off, I would not expect the metadata to be stripped from the note itself - just the published gist. Can this be enabled as an option?

**Comment by timrogers** [comment]
The intent of this option when I built the extension was actually to allow you to use the extension without it adding front-matter (metadata) to your notes at all. I'd be happy to add a separate option to strip the metadata from published gists. I can do that.

**Comment by StephenCoady** [comment]
Makes sense. Thanks!

**Comment by timrogers** [comment]
Reading this again, I think I might have been confused. Would you be able to explain again what behaviour you are seeing, and share a screenshot of your plugin settings?

**Comment by StephenCoady** [comment]
Sure.

Settings:

![Screenshot 2023-09-28 at 10 30 47](https://github.com/timrogers/obsidian-share-as-gist/assets/7307251/3825db79-ed71-4817-a942-9c409abbedf6)

Expected behaviour:
1. CMD+P "Share as private gist..."
2. Gist is created from the note
3. The frontmatter is not included in the published document
4. The note is exactly as it was before publishing

Actual behavior:
1. CM+P "Share as private gist..."
2. Gist is created from the note
3. The frontmatter is not included in the published document
4. The note has been altered locally with the frontmatter removed

Basically, point 4 is IMO unexpected.

**Comment by timrogers** [comment]
Intriguing! Let me take a look.

**Comment by timrogers** [comment]
Fixed in v1.3.1

**Comment by StephenCoady** [comment]
amazing. thank you!
